### DISCUSSION TITLE
Font library for small screens

### DISCUSSION THREAD

**Discussion by Rumidom** [micropython/micropython#14109] [Show and tell]
I needed a smaller font for a project so I made this, I know there are already solutions for converting ttf fonts to use with micropython but I wanted something simpler, this library adds 4 mono-spaced fonts, you'll also have the ability of easily modifying them as bitmaps

https://github.com/Rumidom/micropython_fontlib

**Comment by yvonne910rhodes** [comment]
Hello,

That sounds like a great project! Creating a font library that is simple to use and modify can be incredibly useful for projects with small screens, especially when working with microcontrollers like those compatible with MicroPython [Official Website](https://www.my-milestonecard.com/). Your approach to provide mono-spaced fonts and the ability to modify them as bitmaps offers flexibility for developers looking to customize the display to their needs.

**Comment by robert-hh** [comment]
See also the work of @peterhinch  at https://github.com/peterhinch/micropython-font-to-py
